sunpy/net/jsoc/jsoc.py

```diff
@@ -177,6 +177,14 @@ class JSOCClient(object):
     >>> res.wait(progress=True)   # doctest: +SKIP
     """
 
+    def initialise(self, ser):
```

Comment:
Do we want to keep this given our recent discussions about handling primekeys? I don't really like the injecting of things into the namespace.